app/controllers/api/v1/burgers_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Api::V1::BurgersController < ApiController
+  def show
+    @burger = Burger.all.sample
+    render json: @burger
+  end
```

Comment:
So this action ignores the `:id` that is passed in through params, in favor of returning a random burger? Interesting.